ext/standard/tests/file/bug60120.phpt

```diff
@@ -16,7 +16,7 @@ $php = getenv('TEST_PHP_EXECUTABLE');
 if (!$php) {
     die("No php executable defined\n");
 }
-$cmd = 'php -r "fwrite(STDOUT, $in = file_get_contents(\'php://stdin\')); fwrite(STDERR, $in);"';
+$cmd = 'php -r "\$in = file_get_contents(\'php://stdin\'); fwrite(STDOUT, \$in); fwrite(STDERR, \$in);"';
```

Comment:
I wonder, how this worked before... :(
Also ``'php...`` should be replaced by ``$php . '...``